media/doc/README.WINE

```diff
@@ -78,7 +78,7 @@ dll/win32/iccvid              # Synced to WineStaging-4.0
 dll/win32/ieframe             # Synced to WineStaging-4.18
 dll/win32/imaadp32.acm        # Synced to WineStaging-4.0
 dll/win32/imagehlp            # Synced to WineStaging-4.18
-dll/win32/imm32               # Synced to WineStaging-4.18
+dll/win32/imm32               # Forked at WineStaging-4.18
```

Comment:
Place the implementation in `#ifdef __REACTOS__` instead. Why fork the whole component only for the one small implementation? It makes sense only if there are too many differences with Wine in the whole file.

Reply:
OK.